src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { useContext } from 'react';
+import cn from 'classnames';
+import { TodoMethod, TodosContext } from '../../context/ToDosContext';
+import { SortType } from '../../types/types';
+
+export const Footer: React.FC = () => {
+  const methods = useContext(TodoMethod);
+  const { todos, sorted, setSorted } = useContext(TodosContext);
+
+  const activeTodos = todos.filter(todo => !todo.completed);
```

Comment:
Wrap `activeTodos` in a `useMemo` to avoid unnecessary reprocessing

Reply:
Done!